css/index.css

```diff
@@ -3,28 +3,35 @@
 }
 
 .whole {
-    padding: 10px 3%;
-    margin: auto; 
+    padding: 10px;
+    margin: 10px;
 }
+/* .about {
+    margin-top: 50px;
+} */
 
 header {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding:10px 20px;
+    /* padding:10px; */
     position: fixed; 
     top: 0;
+    max-width: 500% ;
     width: 100%;
+    margin-left: -20px;
```

Comment:
It looks like you currently h ave to set this to offset the `margin: 10px` and `padding: 10px` on the `.whole` div. One way to avoid having to do something like this is to have the `header` sit outside of the `.whole` div:  

```html
<body>
  <header>
   ... header contents ...
  </header>
  <div class="whole">
    <section id ="about">...</section>
    <section id ="skills">...</section>
    <section id ="projects">...</section>
    <section id ="experience">...</section>
    ...etc...
  </div>
</body>
```

Another approach would be to still have `header` inside `.whole` (in case you want some styles to affect both of those parts of the page, but wrap the sections separately so that you can apply the margin and padding just to those:

```html
<body>
    <div class="whole">
        <header>...header contents</header>
        <div class="sections-container">
            <section id ="about">...</section>
            <section id ="skills">...</section>
            <section id ="projects">...</section>
           <section id ="experience">...</section>
           ...etc...
        </div>
    </div>
</body>
```

```css
.whole {
 ...styles that you want to affect both the navbar and the sections (like maybe a common font)
}

.sections-container {
  margin: 10px;
  padding: 10px;
}

/* and now header {} doesn't need margin: -20px at all anymore */
```